Bottom: 5efd454b9a3bc597bb7395352542ae135e2c26ed
Top:    1840970b536d9f7c6ad012b0980d4ba399e89494
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 16:04:49 +0000

Refresh of and-working-again-mostly

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index 94c3973..2206b3a 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -340,7 +340,7 @@ object3d_to_step_fragment (step_file *step, object3d *object, char *part_id, cha
 #undef REV
 }
 
-static void
+void
 object3d_list_export_to_step_assy (GList *objects, char *filename)
 {
   step_file *step;
diff --git a/src/hid/step/object3d_step.h b/src/hid/step/object3d_step.h
index 049fc6b..740937d 100644
--- a/src/hid/step/object3d_step.h
+++ b/src/hid/step/object3d_step.h
@@ -1 +1,2 @@
+void object3d_list_export_to_step_assy (GList *objects, char *filename);
 void object3d_export_to_step (object3d *object, char *filename);
